RELEASING.md

```diff
@@ -34,3 +34,34 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
 
 TODO
+
+## Backporting Guide
+
+Here are the commands we typically run to backport changes to release branches.
```

Comment:
Explain why we backport and when we backport?

I believe we backport security fixes to a dot release (e.g. 1.0.1), small improvements and features to 1.x, and only make backwards-incompatible changes in major versions (basically we follow semver).

Reply:
Should we link out to https://semver.org/spec/v2.0.0.html from here?

Reply:
It's a good idea.